src/platform/user/authentication/components/DowntimeBanner.js

```diff
@@ -5,36 +5,80 @@ import environment from 'platform/utilities/environment';
 import { getStatusFromStatuses } from '../constants';
 
 export default function DowntimeBanners() {
-  const { loading, statuses = [] } = useSelector(
+  const { loading, statuses = [], maintenanceWindows = [] } = useSelector(
     state => state.externalServiceStatuses,
   );
   const dispatch = useDispatch();
   const isLocalhost = useMemo(() => environment.isLocalhost(), []);
+
   useEffect(() => {
     if (!loading && !isLocalhost) {
       dispatch(getBackendStatusAction());
     }
-  }, []); // only on load
+  }, []);
+
+  let downStatus = null;
+  let maintenanceStatus = null;
+  let bannerStatus = [];
+
+  let statusArray = statuses && !isLocalhost ? statuses : [];
+
+  statusArray =
+    statusArray.length === 0 && !isLocalhost
+      ? [{ service: 'mvi', serviceId: 'mvi', status: 'down' }]
+      : statusArray;
+
+  downStatus =
+    statusArray.length > 0 ? getStatusFromStatuses(statusArray) : null;
+
+  if (maintenanceWindows.length > 0) {
+    const checkMaintenanceWindow = () => {
+      const currentTime = new Date();
+      return maintenanceWindows.some(window => {
+        const startTime = new Date(window.start_time);
+        const endTime = new Date(window.end_time);
+        const oneHourBeforeStart = new Date(
+          startTime.getTime() - 60 * 60 * 1000,
+        );
+
+        return currentTime >= oneHourBeforeStart && currentTime <= endTime;
+      });
+    };
+
+    const formattedMaintenanceStatus = () => {
+      const formattedStatus = maintenanceWindows
+        .filter(checkMaintenanceWindow)
+        .map(window => ({
+          csp: window.external_service,
+          status: 'maintenance',
+          startTime: window.start_time,
+          endTime: window.end_time,
+        }));
+      return [...formattedStatus];
+    };
+
+    maintenanceStatus = checkMaintenanceWindow()
+      ? getStatusFromStatuses(formattedMaintenanceStatus())
+      : null;
+  }
+
+  const notNullCombined = (...arrays) =>
+    [].concat(...arrays.filter(Array.isArray));
 
-  // mimics the mvi service error if we don't get an OK response from vets-api
-  const statusArray =
-    statuses || isLocalhost
-      ? []
-      : [{ service: 'mvi', serviceId: 'mvi', status: 'down' }];
-  const bannerStatus = getStatusFromStatuses(statusArray);
-  if (!Object.keys(bannerStatus).length) return null;
-  const { headline, status: alertStatus, message } = bannerStatus;
+  bannerStatus = notNullCombined(downStatus, maintenanceStatus);
 
   return (
     !loading && (
       <div className="downtime-notification row">
         <div className="columns small-12">
-          <div className="form-warning-banner fed-warning--v2">
-            <va-alert visible status={alertStatus}>
-              <h2 slot="headline">{headline}</h2>
-              {message}
-            </va-alert>
-          </div>
+          {bannerStatus.map(({ headline, status, message }, index) => (
```

Comment:
I know I told you to just stack the `va-alerts` however I researched / found out a couple different things since last we spoke:
- Design system says to not stack alerts ([Reference](https://design.va.gov/components/alert#how-to-use-alerts))
- However Riley mentioned that the `statuses` should never have a downtime, when there is scheduled maintenance.